.github/workflows/deploy-gh-pages.yml

```diff
@@ -0,0 +1,81 @@
+# This workflow will do a clean installation of node dependencies, cache/restore them, build the source code and site, and run the deploy script.
+# The deploy script will publish the build to gh-pages.
+# This workflow will be run whenever a new release is published.
+
+# NOTE (pradeep): I'm having this workflow bail out if the release doesn't target v1.2.x. 
+# This is because we don't have a way (yet) to support multiple FDT versions/docs  be deployed at the same time,
+# and hence we choose to deploy only the most stable release.
```

Comment:
In the near/distant future, I'm thinking of having something like [lodash's docs site](https://lodash.com/docs/4.17.15) which runs multiple major releases, and has a extremely simple dropdown at the top to switch between versions.